app/src/redux-resources/analytics/hooks/useCameraAnalytics.ts

```diff
@@ -0,0 +1,186 @@
+import {
+  ANALYTICS_CAMERA_ENABLEMENT_KIND,
+  ANALYTICS_CAMERA_SETTINGS_KIND,
+  ANALYTICS_IMAGE_CAPTURE_KIND,
+  ANALYTICS_LIVE_FEED_KIND,
+  ANALYTICS_PHOTO_ACCESS,
+  useTrackEvent,
+} from '/app/redux/analytics'
+
+import type { RobotType } from '@opentrons/shared-data'
+import type {
+  SOURCE_ROBOT_SETTINGS,
+  SOURCE_RUN_RECORD,
+} from '/app/redux/analytics'
+
+export interface CameraAnalyticsParams {
+  source: typeof SOURCE_RUN_RECORD | typeof SOURCE_ROBOT_SETTINGS
+  robotType?: RobotType
+}
+
+export interface CameraAnalyticsRunRecord extends CameraAnalyticsParams {
```

Comment:
```suggestion
export interface CameraAnalyticsRunRecord {
```
Remember that `extends` effectively says "we want these params plus all the params available on the type we are extending". However, because the `useCameraAnalytics` hook itself takes in the `robotType` and `source`, we don't need the analytic fns to pass this information in - we have direct access to it through the hook. The functions declared in the body of the hook are in what's called the lexical scope of the hook, and have access to that information directly.

This is true of all these types - they don't need to `extend` fortunately.